jenkins/ci.suse.de/openstack-mkcloud.yaml

```diff
@@ -357,6 +357,12 @@
           default:
           description: set to 1 to ping running instances during the upgrade
 
+      - string:
+          name: SUPPORTCONFIG_OPTIONS
```

Comment:
Do we need to expose this in this way?
The default will be set by mkcloud anyway.
And in the rare cases that we need to change this, it can be set using `custom_settings` manually.
Or do you plan to use this variable from regular CI jobs?